_config.yml

```diff
@@ -7,13 +7,14 @@ collections:
   questions:
     output: true
     permalink: /:categories/:name/
+  stories:
+    output: false
 
 sass:
   load_paths:
     - css
     - css-arabic
```

Comment:
Ik bedacht me nog dat het beter zou zijn om dit `css-rtl` te noemen, maar dan heb ik feedback op iets wat ik zelf heb geintroduceerd ;)

Reply:
Ja, liefst zou ik dit ook willen samenvoegen. Dat maakt onderhoud / wijzigingen eenvoudiger.

Reply:
Ik heb sowieso de bestanden hernoemd, samenvoegen is een nice to have en noteer ik voor als er tijd is